modelRepo/deq/sir.deq

```diff
@@ -1,8 +1,8 @@
-let beta = 0.4
-let gamma = 0.04
-let i_initial = 3.0
-let r_initial = 0.0
-let s_initial = 997.0
+param beta = 0.4
+param gamma = 0.04
+param i_initial = 3.0
+param r_initial = 0.0
+param s_initial = 997.0
```

Comment:
To nitpick, I might prefer to see the declaration form be `parameter`, to mirror ESL models. To pick something larger than a nit, I also hesitate to have a model in our `modelRepo` that can't be parsed by our machinery. How would you feel about updating the DEQ parser to support this syntax?

Reply:
> To nitpick, I might prefer to see the declaration form be `parameter`, to mirror ESL models.

Oh right, I somehow had the notion that it was `param` instead of `parameter`. I've changed it to use the latter.

> To pick something larger than a nit, I also hesitate to have a model in our `modelRepo` that can't be parsed by our machinery. How would you feel about updating the DEQ parser to support this syntax?

Alright, I've added that ability as well. But only since you asked so nicely :)